src/main/java/com/minecolonies/api/tileentities/storageblocks/IStorageBlockInterface.java

```diff
@@ -0,0 +1,117 @@
+package com.minecolonies.api.tileentities.storageblocks;
+
+import com.minecolonies.api.colony.buildings.IBuilding;
+import com.minecolonies.api.crafting.ItemStorage;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+import java.util.Map;
+import java.util.function.Predicate;
+
+public interface IStorageBlockInterface
```

Comment:
Missing javadoc on the interface itself.

Reply:
Done.